Makefile

```diff
@@ -154,7 +154,11 @@ integration-api-local:
 .PHONY: cypress-local
 cypress-local:
 	docker build -f ./cypress/Dockerfile  -t cypress:latest .; \
-	docker run -it -e "CYPRESS_baseUrl=https://localhost:7002" --network="host" --rm cypress:latest cypress run --spec cypress/integration/BasicLogin.feature
+	aws-vault exec moj-lpa-dev -- docker run -it -e AWS_ACCESS_KEY_ID -e AWS_SECRET_ACCESS_KEY -e AWS_SESSION_TOKEN -e "CYPRESS_baseUrl=https://localhost:7002" --entrypoint ./node_modules/.bin/cypress-tags --network="host" --rm cypress:latest run -e TAGS='not @SignUp' 
```

Comment:
I would remove `TAGS` from this command line as it's ignored in start.sh anyway.

Reply:
now does the same way as CI